package/devel/trace-cmd/Makefile

```diff
@@ -44,19 +44,6 @@ TARGET_CFLAGS += --std=gnu99 -D_GNU_SOURCE
 define Package/trace-cmd/install
 	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/lib/traceevent/plugins
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/trace-cmd $(1)/usr/bin/
-	$(CP) \
-		$(patsubst %,$(PLUGINS_DIR)/plugin_%.so,$(PLUGINS_MAIN)) \
```

Comment:
Please remove the definition of `PLUGINS_DIR` and `PLUGINS_MAIN` from this package Makefile, they are not used any more.